tools/telemetry-generator/package.json

```diff
@@ -35,7 +35,7 @@
 		"applicationinsights": "^2.4.1"
 	},
 	"devDependencies": {
-		"@fluidframework/build-common": "^2.0.1",
+		"@fluidframework/build-common": "^3.0.0-223236",
```

Comment:
No objection to updating it here, but this wasn't necessary to make things work, right?

Reply:
just got swept up in my regex. i don't think this is a valid for that package